packages/react/package.json

```diff
@@ -23,8 +23,7 @@
   "dependencies": {
     "loose-envify": "^1.1.0",
     "object-assign": "^4.1.1",
-    "prop-types": "^15.6.2",
-    "react-scheduler": "^0.1.0-alpha-1"
```

Comment:
I thought so too, but it results in the scheduler methods (e.g. `cancelScheduledWork`) being embedded in the package, so... I re-added it.

Reply:
If it embeds scheduler methods it means it always imports them regardless. So this is bad by itself (we don't expect them to be in CJS builds). Are they not getting tree shaken due to init time side effects? This might mean our approach with `__UMD__` is not enough and we actually need to fork that file.

Reply:
The approach seems to be working fine. At least, the bundles look good to me. Removing this dep causes troubles though.

There may be a reasonable explanation. I'm not going to dig into it today though. (Holiday.)

I think this PR is good at this point. The deposit was already there so it's at least not new from this PR.